lib/pom.rb

```diff
@@ -223,7 +214,7 @@ def installer.ensure_required_ruby_version_met; end
           specname = File.basename( specfile )
           log "copy to specifications/default: #{specname}"
 
-          spec = Gem::Package.new( Dir[ File.join( cache, "#{g.name}-#{version}*.gem" ) ].first ).spec
+          spec = Gem::Package.new( Dir[ File.join( cache, "#{gem_name}*.gem" ) ].first ).spec
```

Comment:
you might end up including more if cache is not empty e.g. `bin/jruby -S gem` install-ed a newer version